lib/wallet/src/Cardano/Wallet/Write/Tx/Balance.hs

```diff
@@ -76,6 +76,8 @@ import Cardano.Wallet.Primitive.Types.UTxOIndex
     ( UTxOIndex )
 import Cardano.Wallet.Primitive.Types.UTxOSelection
     ( UTxOSelection )
+import Cardano.Wallet.Read.ProtocolParameters
```

Comment:
```suggestion
import Cardano.Wallet.Write.ProtocolParameters
```

Reply:
Thanks @HeinrichApfelmus: this is fixed in 70e564a99f81f1fc96696fa62f8d494754397ecf.